LogicalTypes.md

```diff
@@ -144,6 +169,9 @@ example, there is no requirement that a large number of days should be
 expressed as a mix of months and days because there is not a constant
 conversion from days to months.
 
+The sort order used for `INTERVAL` is `UNSIGNED`, produced by sorting by
+the value of months, then days, then milliseconds with unsigned comparison.
+
 ## Embedded Types
```

Comment:
In the latest commit, I clarified that there isn't a required ordering for embedded types.